numpy/core/src/umath/loops_trig.dispatch.c.src

```diff
@@ -0,0 +1,239 @@
+/*@targets
+ ** $maxopt $werror baseline
+ ** (avx2 fma3) avx512f
+ ** vsx2
+ ** neon_vfpv4
+ **/
+#include "numpy/npy_math.h"
+#include "simd/simd.h"
+#include "loops_utils.h"
+#include "loops.h"
+/*
+ * TODO:
+ * - use vectorized version of Payne-Hanek style reduction for large elements or
+ *   when there's no native FUSED support instead of fallback to libc
+ */
+#if NPY_SIMD_FMA3 // native support
```

Comment:
policy `$werror` didn't break the build after moving the new intrinsics into separate pull-requests.
it seems CI doesn't have any compilation with the GCC flag `-Wundef` enabled.

Reply:
xref #17791